src/test/fuzz/coinscache_sim.cpp

```diff
@@ -135,13 +135,7 @@ struct CacheLevel
     }
 };
 
-/** Class for the base of the hierarchy (roughly simulating a memory-backed CCoinsViewDB).
- *
- * The initial state consists of the empty UTXO set, though coins whose output index
- * is 3 (mod 5) always have GetCoin() succeed (but returning an IsSpent() coin unless a UTXO
- * exists). Coins whose output index is 4 (mod 5) have GetCoin() always succeed after being spent.
```

Comment:
This part `Coins whose output index is 4 (mod 5) have GetCoin() always succeed after being spent` is due to us not erasing these coins with this condition on line 169 below. Since we are not yet not returning spent coins (`// TODO GetCoin shouldn't return spent coins`), we shouldn't remove this line of the comment.

Reply:
[Done](https://github.com/bitcoin/bitcoin/compare/f68e9d8887651828bd99635165fcec43f5cf6d4a..376ba5fcfd72a253b2dfe853781a10eee1af2ccb), thanks